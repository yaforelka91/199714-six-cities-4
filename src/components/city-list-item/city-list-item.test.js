import React from 'react';
import renderer from 'react-test-renderer';
import CityListItem from './city-list-item.jsx';

const city = {
  name: `city 1`,
  coords: [0, 0],
  zoom: 1,
};

describe(`CityListItemSnapshot`, () => {
  it(`should render CityListItem`, () => {
    const tree = renderer.create(
        <CityListItem
          city={city}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
