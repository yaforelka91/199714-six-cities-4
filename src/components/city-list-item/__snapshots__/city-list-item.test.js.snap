// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityListItemSnapshot should render CityListItem 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      city 1
    </span>
  </a>
</li>
`;
